introduction.md

```diff
@@ -1,2 +1,16 @@
 It's me, Byron!
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?